lib/src/track/local/local.dart

```diff
@@ -170,4 +173,32 @@ abstract class LocalTrack extends Track {
     // mark as started
     await start();
   }
+
+  @internal
+  @mustCallSuper
+  Future<bool> publish() async {
```

Comment:
Should rename to `onPublish()` just to be clear that this doesn't actually do the track publishing itself.

Reply:
i agree !